### PR TITLE
PT5290 update geopandas

### PR DESCRIPTION
**Pull request by andher1802**
Updating geopandas to all versions from 0.0.0 to <1.0.0

Items:
* [ ] Implemented (new) unit tests
* [ ] Updated [documentation](sdk.up42.com)

For release:
* [x] Bumped version
* [x] Added changelog

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -64,6 +64,8 @@ mkdocs-redirects = ">=1.2.1"
 pythonpath = "."
 addopts = "--cov --cov-report=html --cov-report=xml --junitxml=out/test_results/junit.xml"
 
+
+
```
Comment (javidq) [inline pyproject.toml:68]: Not needed

File: pyproject.toml [hunk 2]
```diff
@@ -21,7 +21,7 @@ python = ">=3.9, <4"
 requests = "^2.31.0"
 tqdm = "^4.66.0"
 geojson = "3.1.0"
-geopandas = "^0.13.2"
+geopandas = "^0.14.0"
```
Comment (javidq) [inline pyproject.toml:24]: Why can't we set in generic form to anything with this major version?

**Review by javidq — CHANGES_REQUESTED** [review]

File: pyproject.toml (hunk 1 above)
Comment (andher1802) [reply pyproject.toml]: some auto formatting from my IDE, fixed. :)

File: pyproject.toml (hunk 2 above)
Comment (andher1802) [reply pyproject.toml]: setted to >=0.0.0, <1.0.0 https://python-poetry.org/docs/dependency-specification/

**Review by javidq — APPROVED** [review]

**Pull request merged**